types/array.prototype.flatmap/auto.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const noExports: {};
```

Comment:
This file is the one that patches the global array, right? It might be correct to add
```ts
declare global {
  interface Array<T> {
    flatMap<A, B, T extends object | undefined = undefined>(xs: ReadonlyArray<A>, fn: (this: T, x: A, index: number, array: A[]) => B[], thisArg?: T): B[];
  }
}
```
into this file so the global type is actually patched when this file is included in a project.

Reply:
Oh, also a much shorter way to mark the file as a module than
```ts
declare const noExports: {};
export default noExports;
```
is
```ts
export {};
```
and is probably what you'd prefer here.